### PR TITLE
Add the changes made by the new developers

### PR DESCRIPTION
**Pull request by mrchrisadams**
This PR is a summary of the changes made by external developers Gavin worked with over the last couple of months. I have posted it here to make it easier to understand the changes made.

I only had access to the delivered zipfile, and had to run a diff against the main branch of this, ignoring whitespace changes to see what had changed. If you don't ignore whitespace changes it's a bit difficult to see what has changed - if you append `?w=1` to the view below, it's easier to see what has changed, like so:

https://github.com/agentGav/cl8/pull/1/files?w=1

I've used Github Copilot to summarise the changes in this PR

---

> This pull request includes several changes to the `cl8` project, focusing on enhancing the admin interface, updating the CSS for a consistent look and feel, and improving the HTML templates for better user experience. Below are the most important changes grouped by theme:
> 
> ### Admin Interface Enhancements:
> * Added `SendInviteEmailContent` and `PasswordResetEmailContent` models to the admin interface and registered their respective admin classes (`SendInviteEmailContentAdmin`, `PasswordResetEmailContentAdmin`) in `cl8/admin.py`. [[1]](diffhunk://#diff-e055f52b89feabfc33e5a604ed174d500a6ea71ab453e5d7340670bdd33714c9L21-R23) [[2]](diffhunk://#diff-e055f52b89feabfc33e5a604ed174d500a6ea71ab453e5d7340670bdd33714c9R304-R306)
> 
> ### CSS Updates:
> * Updated `cl8/static/css/template.css` to include new styles for various elements such as the body background, login card, buttons, and tags. This ensures a consistent and modern look across the application.
> 
> ### JavaScript Enhancements:
> * Added a script in `cl8/static/js/template.js` to toggle classes on input fields based on their content, improving the user interaction experience.
> 
> ### HTML Template Improvements:
> * Updated `_nav_menu.html` to include a search component and improved layout for better navigation and usability. [[1]](diffhunk://#diff-44921a7e5d7a6f573ed57e9800dd6710f92ac8ac59c919729141cbbe12253a2aR2) [[2]](diffhunk://#diff-44921a7e5d7a6f573ed57e9800dd6710f92ac8ac59c919729141cbbe12253a2aL21-R71) [[3]](diffhunk://#diff-44921a7e5d7a6f573ed57e9800dd6710f92ac8ac59c919729141cbbe12253a2aL59-R93) [[4]](diffhunk://#diff-44921a7e5d7a6f573ed57e9800dd6710f92ac8ac59c919729141cbbe12253a2aL81-R117)
> * Enhanced `_profile.html` to include organization details, social media links, and improved styling for the profile and tag buttons. [[1]](diffhunk://#diff-dbd04f1f2bac8e3696ae0c5b13d0e3ac00c435939539f7259d229d0715326167L2-R3) [[2]](diffhunk://#diff-dbd04f1f2bac8e3696ae0c5b13d0e3ac00c435939539f7259d229d0715326167L13-R23) [[3]](diffhunk://#diff-dbd04f1f2bac8e3696ae0c5b13d0e3ac00c435939539f7259d229d0715326167R40-R64) [[4]](diffhunk://#diff-dbd04f1f2bac8e3696ae0c5b13d0e3ac00c435939539f7259d229d0715326167L72-R73) [[5]](diffhunk://#diff-dbd04f1f2bac8e3696ae0c5b13d0e3ac00c435939539f7259d229d0715326167L85-R86) [[6]](diffhunk://#diff-dbd04f1f2bac8e3696ae0c5b13d0e3ac00c435939539f7259d229d0715326167L133-L153)
> 
> These changes collectively enhance the functionality, appearance, and user experience of the application.

### CODE REVIEW

File: cl8/templates/account/email/account_already_exists_message.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{% extends "account/email/base_message.txt" %}
+{% load i18n %}
+
+{% block content %}{% autoescape off %}{% blocktrans %}You are receiving this email because you or someone else tried to signup for an
+account using email address:
+
+{{ email }}
+
+However, an account using88888888888888888888888888888888888888888888888 that email address already exists.  In case you have
```
Comment (mrchrisadams) [inline cl8/templates/account/email/account_already_exists_message.html:9]: This looks like a typo for email. Easy fix tho.

File: cl8/users/forms.py [hunk 2]
```diff
@@ -1,137 +1,186 @@
...
-        model = Profile
-        fields = [
-            "photo",
-            "name",
-            "phone",
-            "location",
-            "organisation",
-            "bio",
-            "tags",
-            "twitter",
```
Comment (mrchrisadams) [inline cl8/users/forms.py:68]: If this is used on an existing constellation, you'll like need a bit of code to handle this usage of different columns in the database

**Review by mrchrisadams — COMMENTED** [review]
I've added a few very quick notes. This is **_not_** a complete code review

File: cl8/users/forms.py (hunk 2 above)
Comment (agentGav) [reply cl8/users/forms.py:68]: Yes, we've changed to be 'generic social links', esp. as we no longer use or endorse the site formerly known as twitter. It autonames the link based on the root domain (e.g. linkedin, mastodon, etc)

**Comment by mrchrisadams** [comment]
OK, I've had a look over this. I have tests passing locally by skipping some migrations (things you will want to have for future changes going forward), but I don't seem to have access to make the necessary changes to the testing setup to fix it here.

I've made some changes to the config to update the _defaults_ in this repo so we have more meaningful results from the tests. If there is data there in a constellation, will still be some work needed to migrate it, otherwise you'll have to create one from new.

**Pull request merged**